deploy/11_add_bdus_doc_pool.ts

```diff
@@ -0,0 +1,39 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types";
+import type { DeployFunction } from "hardhat-deploy/types";
+import * as constants from "../utils/Constants";
+import { formatAddress, getBdUs, getUniswapFactory } from "../utils/DeployedContractsHelpers";
+import { deployPairOracle, setupStakingContract } from "../utils/DeploymentHelpers";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  console.log("starting deployment: BDUS-DOC");
+
+  if (hre.network.name !== "rsk" && hre.network.name !== "mainnetFork") {
+    throw Error(`Please add support for new network: ${hre.network.name}`);
+  }
+
+  const factory = await getUniswapFactory(hre);
+  const bdus = await getBdUs(hre);
+  const secondaryExternalUsdStable = constants.SECONDARY_EXTERNAL_USD_STABLE[hre.network.name];
+
+  await (await factory.createPair(bdus.address, formatAddress(hre, secondaryExternalUsdStable.address))).wait();
+  await deployPairOracle(hre, "BDUS", secondaryExternalUsdStable.symbol, bdus.address, formatAddress(hre, secondaryExternalUsdStable.address));
+  await setupStakingContract(
```

Comment:
Also add formatAddress to bdus address

Reply:
it's not necessary when we use `getContract` to get the address